dev/com.ibm.ws.transport.http/resources/com/ibm/ws/http/channel/internal/resources/httpchannelmessages.nlsprops

```diff
@@ -141,3 +141,23 @@ duplicateAlias.useraction=Ensure that all host aliases defined as virtual host e
 context.root.changed=CWWKT0028I: Web application moved ({0}): {1}
 context.root.changed.explanation=This message is for informational purposes only.
 context.root.changed.useraction=No action is required.
+
+compression.duplicateType=CWWKT0029W: A duplicate add of the {0} type was attempted.The {1} default configuration is used instead. 
```

Comment:
add a space before the period-

A duplicate add of the {0} type was attempted. The {1} default configuration is used instead.